### DISCUSSION TITLE
LTSP cluster

### DISCUSSION THREAD

**Discussion by BroodTax** [ltsp/ltsp#732] [Q&A] [answered]
Hello! Is clustering possible in the new version of LTSP?

**Comment by alkisg** [comment, marked as answer]
Hi, ltsp-cluster was a tool for LTSP5, that stopped being maintained long before the new LTSP was written. I mean, the latest old LTSP didn't support clustering.

In the easiest form, just splitting e.g. 500 workstations to 5 different LTSP servers, it's rather easy, even if you use LDAP for the accounts.
If you add requirements like "automatic home duplication" then you'll need to manually configure these things, with ceph, zfs snapshost beaming, lsyncd or whatever other suits you; LTSP doesn't automate that part but if you manually configure it, then it can certainly use it.
